openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -523,11 +508,11 @@ OpenAssessment.ResponseView.prototype = {
     fileUpload: function() {
         var view = this;
         var fileUpload = $(".file__upload");
-        fileUpload.addClass("is--disabled");
+        fileUpload.prop('disabled', true);
 
         var handleError = function(errMsg) {
             view.baseView.toggleActionError('upload', errMsg);
-            fileUpload.removeClass("is--disabled");
+            fileUpload.prop('disabled', false);
```

Comment:
I have to look at it again.  

So if I am understanding this correctly, this will only add disabled when you use `$( "input" ).prop( "disabled", true );` ?

I believe I used attr and removeAttr().